daffodil-tdml-processor/src/test/java/org/apache/daffodil/tdml/TestRunnerFactory.java

```diff
@@ -33,9 +34,18 @@
 
   @Test
   public void testPrimaryConstructor() {
-    Runner runner = new Runner(null, testDir, testTdmlFile, true, true, false, NoRoundTrip$.MODULE$,
-        "off", JavaConverters.asScalaBufferConverter(Arrays.asList("daffodil", "ibm")).asScala());
+    URI tdmlUri = Misc.getRequiredResource("org/apache/daffodil/tdml/genericTdml.tdml");
+    scala.util.Right<scala.xml.Elem, String> rightURI = new scala.util.Right<scala.xml.Elem, String>(tdmlUri.toString());
```

Comment:
Is it possible to import scala.util.Right and write this line more simply as...?

    Right<Elem, String> rightURI = new Right<>(tdmlUri.toString());

Java 7 or later allows the diamond (<>) to be used on the right side.